src/adapters/communities-db.ts

```diff
@@ -468,9 +468,9 @@ export function createCommunitiesDBComponent(
     async createCommunity(community: CommunityDB): Promise<Community> {
       const id = randomUUID()
       const query = SQL`
-        INSERT INTO communities (id, name, description, owner_address, private, active)
-        VALUES (${id}, ${community.name}, ${community.description}, ${normalizeAddress(community.owner_address)}, ${community.private || false}, ${community.active || true})
-        RETURNING id, name, description, owner_address, private, active, created_at, updated_at
+        INSERT INTO communities (id, name, description, owner_address, private, active, needs_manual_review)
```

Comment:
This is for the future, right?

Reply:
I will revert all the manual review related changes. It's confusing. In the future we can see these commits if we need them